migrate.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+echo "\n\nRenaming files from .jsx to .js"
+find "$1" -name "*.jsx" -exec sh -c 'mv "$1" "${1%.jsx}.js"' _ {} \;
+
+echo "\n\nResolving relative imports"
+jscodeshift -t transforms/resolve-relative-imports.js --extensions js $1
+
+echo "\n\nConverting Helmet imports"
+jscodeshift -t transforms/convert-helmet-import.js --extensions js $1
+
+echo "\n\nConverting Ad imports"
+jscodeshift -t transforms/convert-ad.js --extensions js $1
+
+echo "\n\nConverting Prefetch imports"
+jscodeshift -t transforms/convert-prefetch.js --extensions js $1
+
+echo "\n\nConverting to Radium"
+jscodeshift -t transforms/convert-to-radium.js --extensions js $1
+
+echo "\n\nRemoving Gridiron imports"
+jscodeshift -t transforms/remove-gridiron-import.js --extensions js $1
+
+echo "\n\nRemoving Stilr stylesheets"
+jscodeshift -t transforms/remove-stilr.js --extensions js $1
+
+echo "\n\nMarking index files for deletion"
+for line in $(cat empty_indexes.txt); do
+	if [ ! -z "$line" ]; then
+		if [ -f "$line" ]; then
+			mv -i "${line}" "${line/index.js/delete.index.js}"
```

Comment:
That might work too, but the way I saw it explained was ${var/Pattern/Replacement}
http://unix.stackexchange.com/questions/102647/how-to-rename-multiple-files-in-single-command-or-script-in-unix